client/cpu_benchmark.h

```diff
@@ -32,9 +32,9 @@ extern bool benchmark_time_to_stop(int which);
 // separate different compilations of whetstone.cpp which will utilize
 // various ARM fp features ie neon, vfp, or "normal"
 #ifdef ANDROID
-#ifdef ANDROID_NEON
+#ifdef ANDROID_NEON1
```

Comment:
Where did you get this macro? I believe it's a wrong one ane using it you just disable a part of functionality that should be there